recipe/conda_build_config.yaml

```diff
@@ -11,4 +11,21 @@ channel_priority:
 tiledb:
 - '2.15.*'
 - '2.16.*'
-
+numpy:
+  - 1.20   # [not (osx and arm64) and not win]
+  - 1.20
+  - 1.20
+  - 1.21
+  - 1.23
+python:
+  - 3.7.* *_cpython   # [not (osx and arm64) and not win]
```

Comment:
Note that this is also building a linux-64 Python 3.7 build in addition to the cross-platform Python 3.11 builds